canonical-web-and-design/ubuntu.com
# Create the /kubeconeurope2020 page

## Done

- Create the /kubeconeurope2020 page
- **Note - need to test the livechat button on staging???**

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubeconeurope2020
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1qZGlO8ecig760hXn-8DG3fw52LNHfTM7WKwAHu_DpXc/edit#heading=h.njb3ct3th6b2), test the form


## Issue / Card

Fixes #8039
